### PR TITLE
Update notation for filenames

### PR DESCRIPTION
**Pull request by danielgruesso**
## Description

Update notation for filenames to use all caps to match style elsewhere and aid translations.

### CODE REVIEW

File: src/content/chainlink-nodes/v1/configuration.mdx [hunk 1]
```diff
@@ -272,7 +272,7 @@ _none_ - Disables backups.
 `lite` - Dumps small tables including configuration and keys that are essential for the node to function, which excludes historical data like job runs, transaction history, etc.
 `full` - Dumps the entire database.
 
-It will write to a file like `$ROOT/backup/cl_backup_<VERSION>.dump`. There is one backup dump file per version of the Chainlink node. If you upgrade the node, it will keep the backup taken right before the upgrade migration so you can restore to an older version if necessary.
+It will write to a file like `$ROOT/backup/cl_backup_VERSION.dump`. There is one backup dump file per version of the Chainlink node. If you upgrade the node, it will keep the backup taken right before the upgrade migration so you can restore to an older version if necessary.
```
Comment (thedriftofwords) [inline src/content/chainlink-nodes/v1/configuration.mdx:275]: Source: https://github.com/smartcontractkit/chainlink/blob/develop/core/config/docs/core.toml#L47

File: src/content/chainlink-nodes/v1/node-config.mdx [hunk 2]
```diff
@@ -190,7 +190,7 @@ _none_ - Disables backups.
 `lite` - Dumps small tables including configuration and keys that are essential for the node to function, which excludes historical data like job runs, transaction history, etc.
 `full` - Dumps the entire database.
 
-It will write to a file like `'Dir'/backup/cl_backup_<VERSION>.dump`. There is one backup dump file per version of the Chainlink node. If you upgrade the node, it will keep the backup taken right before the upgrade migration so you can restore to an older version if necessary.
+It will write to a file like `'Dir'/backup/cl_backup_VERSION.dump`. There is one backup dump file per version of the Chainlink node. If you upgrade the node, it will keep the backup taken right before the upgrade migration so you can restore to an older version if necessary.
```
Comment (thedriftofwords) [inline src/content/chainlink-nodes/v1/node-config.mdx:193]: Source: https://github.com/smartcontractkit/chainlink/blob/develop/core/config/docs/core.toml#L47

File: src/content/chainlink-nodes/v1/node-config.mdx [hunk 3]
```diff
@@ -5138,8 +5138,8 @@ If you are using FixedPriceEstimator:
 
 If you are using BlockHistoryEstimator (default for most chains):
 
-- With gas bumping disabled, it will submit all transactions with `feecap=PriceMax` and `tipcap=<calculated using past blocks>`
-- With gas bumping enabled (default for most chains) it will submit all transactions initially with `feecap = ( current block base fee * (1.125 ^ N) + tipcap )` where N is configurable by setting `EVM.GasEstimator.BlockHistory.EIP1559FeeCapBufferBlocks` but defaults to `gas bump threshold+1` and `tipcap=<calculated using past blocks>`
+- With gas bumping disabled, it will submit all transactions with `feecap=PriceMax` and `tipcap=CALCULATED_USING_PAST_BLOCKS`
```
Comment (thedriftofwords) [inline src/content/chainlink-nodes/v1/node-config.mdx:5141]: Source: https://github.com/smartcontractkit/chainlink/blob/5bcd414622294d8142e6cfff08f8c1b0c2dc72df/core/config/docs/chains-evm.toml#L191

File: src/content/chainlink-nodes/v1/node-config.mdx [hunk 4]
```diff
@@ -5138,8 +5138,8 @@ If you are using FixedPriceEstimator:
...
+- With gas bumping enabled (default for most chains) it will submit all transactions initially with `feecap = ( current block base fee * (1.125 ^ N) + tipcap )` where N is configurable by setting `EVM.GasEstimator.BlockHistory.EIP1559FeeCapBufferBlocks` but defaults to `gas bump threshold+1` and `tipcap=CALCULATED_USING_PAST_BLOCKS`
```
Comment (thedriftofwords) [inline src/content/chainlink-nodes/v1/node-config.mdx:5142]: https://github.com/smartcontractkit/chainlink/blob/5bcd414622294d8142e6cfff08f8c1b0c2dc72df/core/config/docs/chains-evm.toml#L192

long line, right scroll all the way to end

**Review by dwightjl — APPROVED** [review]

**Comment by dwightjl** [comment]
I opened a ticket to make the changes in the Chainlink repo.

**Pull request merged**